sklearn/ensemble/tests/test_forest.py

```diff
@@ -1811,6 +1811,51 @@ def test_round_samples_to_one_when_samples_too_low(class_weight):
     forest.fit(X, y)
 
 
+@pytest.mark.parametrize("boostrap", [True, False])
+@pytest.mark.parametrize("ForestClass", FOREST_CLASSIFIERS_REGRESSORS.values())
+def test_estimators_samples(ForestClass, bootstrap):
+    # Check that format of estimators_samples_ is correct and that results
+    # generated at fit time can be identically reproduced at a later time
+    # using data saved in object attributes.
+    X, y = make_hastie_10_2(n_samples=200, random_state=1)
+    est = ForestClass(
+        n_estimators=10,
+        max_samples=0.5,
```

Comment:
I don't think we can set max_samples when bootstrap is False. It raises a ValueError.

Reply:
Ah right you are. Let me fix the test